sox.features.info.json

```diff
@@ -431,7 +431,8 @@
             "extended_description": "Adds a button next to links to posts on the same site that expand to show the post inline",
```

Comment:
Could you update this description as well, to say it now works with posts on any other SE site?

Reply:
Ok, done!